instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py

```diff
@@ -73,21 +73,23 @@ def started(self, event: monitoring.CommandStartedEvent):
             span = self._tracer.start_span(name, kind=SpanKind.CLIENT)
             if span.is_recording():
                 span.set_attribute("component", DATABASE_TYPE)
-                span.set_attribute("db.type", DATABASE_TYPE)
-                span.set_attribute("db.instance", event.database_name)
+                span.set_attribute("db.system", DATABASE_TYPE)
+                span.set_attribute("db.name", event.database_name)
                 span.set_attribute("db.statement", statement)
                 if event.connection_id is not None:
                     span.set_attribute("net.peer.name", event.connection_id[0])
                     span.set_attribute("net.peer.port", event.connection_id[1])
 
                 # pymongo specific, not specified by spec
-                span.set_attribute("db.mongo.operation_id", event.operation_id)
-                span.set_attribute("db.mongo.request_id", event.request_id)
+                span.set_attribute(
```

Comment:
I would remove these attributes as they are not in the semantic conventions.

Reply:
@toumorokoshi also expressed the same concern on this attrs. I will remove them.